### ISSUE TITLE
Export latent state for dynamic version

### ISSUE THREAD

**Issue by WenHanGao** [keyATM/keyATM#188]
## Requested feature

Allows the latent state of each time step to be exported for customise plot.
```r
# Example
value_figure(fig_timetrend)
```
```r
# Maybe one more column called latent state
## # A tibble: 290 × 5
##    time_index Topic             Lower  Point  Upper
##         <int> <chr>             <dbl>  <dbl>  <dbl>
##  1       1789 1_Government     0.100  0.113  0.125 
##  2       1789 2_Congress       0.154  0.182  0.211 
##  3       1789 3_Peace          0.0441 0.0618 0.0800
##  4       1789 4_Constitution   0.215  0.232  0.254 
##  5       1789 5_ForeignAffairs 0.121  0.148  0.173 
##  6       1793 1_Government     0.209  0.247  0.287 
##  7       1793 2_Congress       0.0207 0.0699 0.0991
##  8       1793 3_Peace          0.0574 0.0876 0.112 
##  9       1793 4_Constitution   0.257  0.312  0.388 
## 10       1793 5_ForeignAffairs 0.0522 0.0819 0.112 
```
## Use case

Useful for state transition interpretation.

**Comment by Shusei-E** [comment]
Hi, thanks for using our package and sorry for the late response. Could you provide us with a little more details of your use case? The main goal of keyATM is to avoid post-hock interpretation of latent topics, which is why we ask users to set keywords for each topic. We can include latent HMM states in our output, but I'm a bit worried it would confuse users, because they need to interpret latent states and latent states are not our main quantity of interest.

**Comment by WenHanGao** [comment]
One useful case I can think of is when modelling financial corpus across different market periods. Hence, the state switching mechanism in the dynamic version may be able to surface how topic distributions change due to a change in the state of the market. In essence, since these estimates are already computed, it is up to the user to make use of them. Thank you.

**Comment by Shusei-E** [comment]
Hi @WenHanGao, your suggestion is implemented in v0.5.0 (it's on CRAN). Sorry for taking too much time.

Please check [this](https://keyatm.github.io/keyATM/articles/pkgdown_files/Plot.html) for the actual use case.
Basically, `values_fig` includes the `state_id` column.
